dev/ruby-compose.yml

```diff
@@ -0,0 +1,19 @@
+# imap-backup built with different ruby versions for testing
+version: "3"
+
+services:
+  imap-backup:
+    build:
+      context: ../
+      dockerfile: dev/container/Dockerfile
+      args:
+      - RUBY_VERSION
+    image: imap-backup:${RUBY_VERSION}
+    user: $(id -u)
```

Comment:
Using `$(id -u)` gives an error with docker-compose v1. After altering it to `${ID} I ran as follows:

```
ID=$(id -u) RUBY_VERSION=3.2.2 docker-compose -f dev/ruby-compose.yml build
ID=$(id -u) RUBY_VERSION=3.2.2 docker-compose -f dev/ruby-compose.yml up
```

This gave me the following error:
```
imap-backup_1  | /usr/local/lib/ruby/3.2.0/rubygems.rb:263:in `find_spec_for_exe': can't find gem imap-backup (>= 0.a) with executable imap-backup (Gem::GemNotFoundException)
imap-backup_1  |        from /usr/local/lib/ruby/3.2.0/rubygems.rb:282:in `activate_bin_path'
imap-backup_1  |        from /usr/local/bundle/bin/imap-backup:25:in `<main>'
dev_imap-backup_1 exited with code 1
```